recipes/prgenv-gnu/24.11/gh200/environments.yaml

```diff
@@ -0,0 +1,46 @@
+gcc-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+  mpi:
+      spec: cray-mpich@8.1.30
+      gpu: cuda
+  unify: true
+  specs:
+  - boost +chrono +filesystem +iostreams +mpi +python +regex +serialization +shared +system +timer
+  - cmake
+  - fftw
+  - fmt
+  - gsl
+  - hdf5+hl
+  - kokkos +aggressive_vectorization ~alloc_async +cuda_constexpr +cuda_lambda ~cuda_relocatable_device_code ~cuda_uvm cxxstd=17 +openmp +pic +serial +shared +tuning +wrapper
+  - kokkos-kernels +blas +cublas +cusparse +cusolver +execspace_cuda +execspace_openmp +execspace_serial +lapack +memspace_cudaspace +openmp scalars=float,double,complex_float,complex_double +serial +shared +superlu
+  - kokkos-tools +mpi +papi
+  - netlib-scalapack
+  - lua
+  - libtree
+  - lz4
+  - meson
+  - ninja
+  - openblas threads=openmp
+  - osu-micro-benchmarks@5.9
+  - papi ~cuda
+  - python@3.12
+  - zlib-ng
+  # add GPU-specific packages here, for easier comparison with mc version
+  - nccl
+  - nccl-tests
+  - cuda@12.6
+  - aws-ofi-nccl@git.v1.9.2-aws=1.9.2
```

Comment:
Note: I've updated this to use a fixed version so that we know what we're getting. In addition, @teojgo recommended using 1.9.2 as that's what HPE recommends. Unfortunately 1.9.2 isn't yet available in upstream spack (I'm adding them in https://github.com/spack/spack/pull/47717, but that won't be available until 0.24 or 1.0).

Reply:
I would prefer a version preference and not a requirement, otherwise it is hard to test a new release, whether it would work too.
However I think that there should be version preference in the cluster config, because in the recipe you should only care whether you want nccl or not. You should not care too much, to find the current best know working version.

Reply:
https://github.com/eth-cscs/alps-cluster-config/pull/18